src/modules/sync/hooks/use-default-sync-tree.tsx

```diff
@@ -3,19 +3,10 @@ import { useMemo } from 'react';
 import { TreeNode } from 'src/components/tree-view';
 import { SYNC_OPTIONS } from 'src/constants';
 
-export const useDefaultSyncTree = ( type: 'push' | 'pull' ): TreeNode[] => {
+export const useDefaultSyncTree = ( _type: 'push' | 'pull' ): TreeNode[] => {
```

Comment:
Why won't we remove it?
```suggestion
export const useDefaultSyncTree = (): TreeNode[] => {
```

Reply:
Done https://github.com/Automattic/studio/pull/1596/commits/c25b357f1de17169347e8a9854e4b152c390e910
https://github.com/Automattic/studio/pull/1596/commits/180b315eb2c0ba40379c9b3385fea96938483beb